### PR TITLE
Updated lanes

### LINKED ISSUE #88: Render 10 books per lane on the home page
And make it two rows of 5 books for desktop view.

### PR DESCRIPTION
**Pull request by EdwinGuzman**
This fixes #88 
Also removed ids from SVG icons to avoid accessibility "duplicate id" on the same page errors.

### CODE REVIEW

File: templates/bookSocialMedia.html.template [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <a href="https://www.facebook.com/sharer/sharer.php?u=https://subwaylibrary.com{{url}}&display=popup"
       target="_blank" onclick="gap('send', 'event', 'ShareBook', 'Facebook');">
       <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 500" preserveAspectRatio="xMidYMid meet" role="img" aria-labelledby="brooklyn-public-library-logo" class="fb-icon">
-        <title id="share-on-face-book">Share on Face Book</title>
+        <title>Share on Face Book</title>
```
Comment (aslagle) [inline templates/bookSocialMedia.html.template]: isn't facebook one word?

File: about.html.template [hunk 2]
```diff
@@ -102,7 +102,7 @@
 
             <span>
               <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 338.97883 201.02788" preserveAspectRatio="xMidYMid meet" class="icon" role="img" aria-labelledby="new-york-state-logo" class="nys-logo">
-                <title id="new-york-state-logo">New York State</title>
```
Comment (aslagle) [inline about.html.template]: all these ids are referenced in the aria-labelledby in the svg. I think they are still needed?

Comment (aslagle) [inline about.html.template]: is there another copy of that id somewhere else?

File: templates/bookSocialMedia.html.template (hunk 1 above)
Comment (EdwinGuzman) [inline templates/bookSocialMedia.html.template]: @ricardoom is that for screenreaders to read it correctly?

File: about.html.template (hunk 2 above)
Comment (EdwinGuzman) [inline about.html.template]: You're right... that's a problem. That happened on the About page for the svg logos. @ricardoom any idea how to fix?

@aslagle I'll revert that change and deal with that in another issue instead.

File: templates/bookSocialMedia.html.template (hunk 1 above)
Comment (ricardoom) [inline templates/bookSocialMedia.html.template]: fixes an a11y error thrown by aXe....

**Pull request merged**